core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { NgClass, NgIf } from '@angular/common';
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ButtonModule } from 'primeng/button';
+import { CardModule } from 'primeng/card';
+import { ScrollPanelModule } from 'primeng/scrollpanel';
+
+import {
+    TemplateBuilderBoxComponent,
+    TemplateBuilderBoxSize
+} from './template-builder-box.component';
+
+@Component({
+    selector: 'dotcms-host-component',
+    template: ` <dotcms-template-builder-box [size]="size"> </dotcms-template-builder-box>`
+})
+class TemplateBuilderBoxHostComponent {
+    size = 'large';
+}
+
+describe('TemplateBuilderBoxComponent', () => {
```

Comment:
We should use jest as we discussed, you need to import `describe, it, expect` from `@jest/globals`

Reply:
If this is running it means our jest config allow us to use this without explicit imports.